dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/security/config/DhisWebApiWebSecurityConfig.java

```diff
@@ -467,6 +472,11 @@ private void configureAccessRestrictions(
             }
 
             authorize
+
+                .antMatchers( "/impersonate" ).hasAnyAuthority( "ALL", "F_IMPERSONATE_USER" )
```

Comment:
I wonder if maybe we should not give this authority to "ALL" users by default?  Maybe it's something that NEEDS to be explicitly granted?

Reply:
Decided to instead implement a dhis.conf setting to disable the feature entirely (or optionally restrict it by IP)